public/splash.html

```diff
@@ -0,0 +1,138 @@
+<!-- Splash screen built upon vanilla html, js and css -->
+<html>
+  <head>
+    <script defer type="text/javascript">
+      // defer is for executing js before the page shows up
+      const queryString = window.location.search;
+      const urlParams = new URLSearchParams(queryString);
+
+      const primary = urlParams.get("primary");
+      const background = urlParams.get("background");
+      const display = urlParams.get("display");
+      const version = urlParams.get("version");
+
+      const root = document.documentElement;
+
+      // Notice: when passing url parameters, `#` is rstriped for unicode issues
+      root.style.setProperty("--primary", `#${primary}`);
+      root.style.setProperty("--background", `#${background}`);
+      root.style.setProperty("--display", `#${display}`);
+
+      window.onload = function () {
+        document.getElementById("version").innerHTML = `version ${version}`;
```

Comment:
```suggestion
        document.getElementById("version").innerHTML = `Ver. ${version}`;
```
I think it would be better to unify the notation with the title bar.